contracts/cw3-flex-multisig/src/contract.rs

```diff
@@ -1111,10 +1108,10 @@ mod tests {
             .unwrap();
         assert_eq!(
             res.custom_attrs(0),
-            &[
-                attr("action", "execute"),
-                attr("sender", SOMEBODY),
-                attr("proposal_id", proposal_id.to_string()),
+            [
```

Comment:
Wow, this works over arrays as well? Good stuff.

Reply:
Oh, yeah. Sometimes if you implement a trait like `PartialEq` for type `T`, it turns out some generic implementation for `[T; 3]`or `Vec<T>` or something kicks in from Rust's `std` and cool things happen.